wallet/src/de/schildbach/wallet/ui/main/WalletTransactionsFragment.java

```diff
@@ -443,16 +443,15 @@ private void updateSyncState(BlockchainState blockchainState) {
 
             if (percentage == 0) {
                 syncing += "…";
+                syncingText.setText(syncing);
             } else {
                 SpannableStringBuilder str = new SpannableStringBuilder(syncing + " " + percentage + "%");
                 int start = syncing.length() + 1;
                 int end = str.length();
                 str.setSpan(new android.text.style.StyleSpan(android.graphics.Typeface.BOLD), start, end,
                         Spannable.SPAN_EXCLUSIVE_EXCLUSIVE);
-                syncing = str.toString();
+                syncingText.setText(str);
```

Comment:
Have to pass `SpannableStringBuilder` to the `setText`, otherwise bold won't work